client/src/components/reviews/review_form/04.1_IndividualCharacteristic.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import ShowRatingDesc from './04.2_ShowRatingDesc.jsx';
+
+const characteristicDescription = {
+  Size: ['A size too small', '1/2 a size too small', 'Perfect', '1/2 a size too big', 'A size too wide'],
+  Width: ['Too narrow', 'Slightly narrow', 'Perfect', 'Slightly wide', 'Too wide'],
+  Comfort: ['Uncomfortable', 'Slightly uncomfortable', 'Ok', 'Comfortable', 'Perfect'],
+  Quality: ['Poor', 'Below average', 'What I expected', 'Pretty great', 'Perfect'],
+  Length: ['Runs short', 'Runs slightly short', 'Perfect', 'Runs slightly long', 'Runs long'],
+  Fit: ['Runs tight', 'Runs slightly tight', 'Perfect', 'Runs slightly long', 'Runs long']
+};
+
+const IndividualCharacteristic = (props) => {
+
+  const id = props.value.details.id;
+  const characteristic = props.value.characteristic;
+  const [description, setDescription] = useState({});
+
+  const handleCharacteristicRating = (e) => {
+    const ratingDesc = e.target.value.split('-')[0];
```

Comment:
I see you're adding a lot of metadata onto the html element and then extracting it when an event is triggered. You might consider using data attributes to do this a bit more cleanly. Then you could have a bunch of attributes, like data-description, data-id, data-rating, etc., and you can derive the values with something like e.target.dataset.description. Alternatively, you could put something unique on the DOM, like an id, and have your data stored in an object with the id as the key. More generally, and I haven't dug super deep into what exactly the code is doing here, I would figure out what functionality you need and do some work up front to structure (or restructure perhaps) your data to make your life as easy as possible.

Reply:
Hey @chefferson 
This is a very good point. Functionally it works, but it doesn't make sense from an accessibility / document structure perspective. 

I have refactored so that there is only the data that is needed at the element level. This is a good opportunity to ensure that the data I add is enough for what I need to do, and nothing more.